packages/flutter/test/widgets/tap_region_test.dart

```diff
@@ -59,48 +59,48 @@ void main() {
       await gesture.removePointer();
     }
 
-    expect(clickedOutside, isEmpty);
+    expect(tappedOutside, isEmpty);
 
     await click(find.text('No Group'));
     expect(
-        clickedOutside,
+        tappedOutside,
         unorderedEquals(<String>{
           'Group 1 A',
           'Group 1 B',
         }));
```

Comment:
nit: extra indentation and missing comma before ending paranthesis
(here and in several places below)